backend/src/entities/Node.ts

```diff
@@ -0,0 +1,13 @@
+import { Property } from './Property';
+import { NodeDescriptor } from './NodeDescriptor';
+
+/**
+ * Represents a node of a graph with additional information.
+ */
+export interface Node extends NodeDescriptor {
+  id: number;
```

Comment:
Doesn't the `NodeDescriptor` that this type inherits from already specify the `id` member? Why is this duplicated here?